client/state/automated-transfer/selectors.js

```diff
@@ -68,3 +69,21 @@ export const isEligibleForAutomatedTransfer = compose(
 	getEligibilityStatus,
 	getEligibility
 );
+
+/**
+ * Determine if eligibility warnings or holds exist for
+ * a site other than a business plan hold.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} False if there is one and only one hold/warning referring to missing business plan
+ */
+export const hasEligibilityMessagesOtherThanBusiness = ( state, siteId ) => {
```

Comment:
this is a very specific name and it makes me wonder if it's too specific. what is the UI-centric thing we are trying to infer from the state? do we really care about whether or not a business plan is holding us back? what if our _actual_ goal is knowing if we can start the transfer process or if we have blocking holds.

```js
export const hasBlockingEligibilityHolds = …
```